### ISSUE TITLE
The operation on how to extract the beaches_code I need

### ISSUE THREAD

**Issue by barwin814** [jpalomav/SAET_master#3]
Hi, I'm a new college student and I'm looking for some basic instruction.
I want to use this tool to process the coastlines of continents other than Europe, and to do so I need to change the scope, so what do I need to pay attention to when modifying the beach.shp file? When I do the processing, I generate a temp file with a bb300.shp file in it, do I just need to simply put this file into the aux_data folder, and then I can use it as a tool to process the coastlines of continents other than Europe.

**Comment by jpalomav** [comment]
Hi. To use SAET with other beaches outside of Europe, you just have to modify the file "beaches.shp". You can use any GIS software to create new polygons of beaches. The only mandatory requirement is that value in the field "BEACH_CODE" for this new polygon must be unique. I hope this information is useful to you.
Best regards,
SAET team

**Comment by barwin814** [comment]
Certainly, I am grateful for your gratitude. You have rectified the issue and achieved a desirable coastline. My sincerest thanks to you once again.



---Original---
From: ***@***.***&gt;
Date: Fri, Jul 26, 2024 19:39 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [jpalomav/SAET_master] The operation on how to extract thebeaches_code I need (Issue #3)




 
Hi. To use SAET with other beaches outside of Europe, you just have to modify the file "beaches.shp". You can use any GIS software to create new polygons of beaches. The only mandatory requirement is that value in the field "BEACH_CODE" for this new polygon must be unique. I hope this information is useful to you.
 Best regards,
 SAET team
